modP2pImpl/src/org/aion/p2p/impl1/P2pMgr.java

```diff
@@ -198,15 +201,13 @@ public void run() {
                         });
             }
 
-            int pNum = Runtime.getRuntime().availableProcessors();
-
-            for (int i = 0; i < (pNum << 1); i++) {
+            for (int i = 0; i < WORKER; i++) {
```

Comment:
the number of send processes used to be double the number of receive processes, what's the meaning behind having them equal now?

Reply:
each worker thread represents a lane to deal with the peer connection belong to this lane. Cause now the lane is fixed number, then the thread number should be fixed.